### DISCUSSION TITLE
Support importing/exporting keys from/to Paserk

### DISCUSSION THREAD

**Discussion by not-my-profile** [panva/paseto#25] [Ideas]
In order to use Paseto for my application I need cross-language interoperability. However the only two Paseto v4 implementations for Python enforce [Algorithm Lucidity](https://github.com/paseto-standard/paseto-spec/blob/master/docs/02-Implementation-Guide/03-Algorithm-Lucidity.md), meaning they currently cannot be easily used in conjunction with this library, since it doesn't support Paserk.

So it would be great if this library could support importing/exporting keys from/to [Paserk](https://github.com/paseto-standard/paserk) :)

To be specific I'm talking about the `local`, `public` and `secret` PASERK types from [this  matrix](https://github.com/paseto-standard/paserk#paserk).

**Comment by panva** [comment]
https://github.com/panva/paseto/releases/tag/v3.1.0
